helm/cluster-autoscaler-app/values.yaml

```diff
@@ -50,6 +50,11 @@ clusterAutoscalerResources:
     cpu: 100m
     memory: 400Mi
 
+vpa:
+  minAllowed:
```

Comment:
Which cluster are you seeing OOMs ? 

i guess is a big cluster because the ones i looked at are using much less resources than this minimum

Also, should the `minimum` here match the `requests` ? otherwise it will be started with 400Mi of ram and then scale down to 300

Reply:
Adjusted the default settings to be equal to the requests